services/bettersocial/fcmToken/sendChatNotification.js

```diff
@@ -0,0 +1,20 @@
+const {messaging} = require('firebase-admin');
+const FcmTokenFunction = require('../../../databases/functions/fcmToken');
+const {FcmToken, User} = require('../../../databases/models');
+const UsersFunction = require('../../../databases/functions/users');
+
+const sendChatNotification = async (userTargetId, payload) => {
+  const signedUser = await UsersFunction.findSignedUserId(User, userTargetId);
+  const userTargetToken = await FcmTokenFunction.findTokenByUserId(FcmToken, signedUser);
+  if (userTargetToken) {
+    try {
+      messaging()
+        .sendToDevice(userTargetToken?.token, payload)
+        .then(() => {});
+    } catch (error) {
+      console.log('error send chat notifications', error);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Error handling here is done by simply logging the error to the console. This might not be sufficient for production code. Consider throwing the error or handling it in a way that's appropriate for your application's error handling strategy.
```diff
-      console.log('error send chat notifications', error);
+      throw error;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->